### PR TITLE
jquery deprecated from the project

### LINKED ISSUE #419: Deprecate Jquery
Jquery was included by default in Ember.js. Though it has served its purpose well, current state of ember and browsers no longer needs Jquery. Ember also has officially [deprecated Jquery](https://github.com/emberjs/rfcs/blob/master/text/0386-remove-jquery.md), we also wish to do the same.

Task: Remove all usages of jQuery throughout the project. One way to get started is to remove `@ember/jquery` package from package.json and re-rerun `yarn install` and check for errors due to absence of Jquery and fix them with suitable Vanilla JS / Ember code.

### PR DESCRIPTION
**Pull request by farhan787**
fixes #419

### CODE REVIEW

**Comment by farhan787** [comment]
@abhishek97 would you mind continuing with this PR, I've closed the previous PR. Now, `jquery` is totally deprecated and `ember-ajax` is also replaced with `ember-fetch`.

**Comment by abhishek97** [comment]
This pull request **fixes 1 alert** when merging aa9ab5f41405a2a5e66dcc9d986e706c3240c8cd into 45c631056dae827f0ebf392bcfdfdb8b73077556 - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/codingblocks.online.projectx/rev/pr-7a10052ceafd992b5f2f4e68cee0261b2c4db0e8)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by farhan787** [comment]
@abhishek97 I've installed the `LGTM GitHub` App for this repository. Is it good now ?

File: app/services/hb-api.js [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
-    currentUser: service(),
-    host: env.hackApiHost,
-    contentType: 'application/json; charset=utf-8',
-    namespace: '/api',
-    headers: computed('currentUser.user.hackJwt', function () {
+export default {
+  request: function(url, object) {
+    const options = {
+      session: service(),
+      currentUser: service(),
```
Comment (jatinkatyal13) [inline app/services/hb-api.js]: I don’t think fetch library expects session and currentUser !

File: app/services/hb-api.js [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+import fetch from 'fetch';
 
-export default AjaxService.extend({
-    session: service(),
-    currentUser: service(),
-    host: env.hackApiHost,
-    contentType: 'application/json; charset=utf-8',
-    namespace: '/api',
-    headers: computed('currentUser.user.hackJwt', function () {
+export default {
```
Comment (jatinkatyal13) [inline app/services/hb-api.js]: This isn’t even a service. This should extend `@ember/service`

```
export default Service.extend({
    ....
})
```

File: config/environment.js [hunk 3]
```diff
@@ -69,8 +69,8 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.apiHost = 'http://localhost:3000'
-    ENV.publicUrl = 'http://localhost:4200';
+    ENV.apiHost = 'http://127.0.0.1:3000'
```
Comment (jatinkatyal13) [inline config/environment.js]: Unnecessary

File: app/pods/components/carousel-cards/component.js [hunk 4]
```diff
@@ -2,14 +2,13 @@ import Component from '@ember/component';
...
 import { action } from '@ember/object'
-import $ from 'jquery';
 
 class carouselCard {
   constructor (el, cssConf = {}) {
     this.element = el
     this.translateX = 0
     this.translateY = 0
-    $(this.element).css(cssConf)
+    this.element.setAttribute('class', cssConf)
```
Comment (jatinkatyal13) [inline app/pods/components/carousel-cards/component.js]: cssConf are css attributes and not classes, like
```
{
    display: none
}
```

File: app/pods/components/carousel-cards/component.js [hunk 5]
```diff
@@ -25,21 +24,26 @@ class carouselCard {
   }
 
   showContent () {
-    $(this.element).find('*').removeClass('display-none')
-    $(this.element).find('*').addClass('display-block')
+    this.element.querySelectorAll(this.element).childNodes.classList.remove('display-none')
```
Comment (jatinkatyal13) [inline app/pods/components/carousel-cards/component.js]: `Failed to execute 'querySelectorAll' on 'Element': '[object HTMLElement]' is not a valid selector.`

File: app/pods/components/hls-player/component.js [hunk 6]
```diff
@@ -46,13 +44,22 @@ export default Component.extend(KeyboardShortcuts, {
     if (this.copySrc === this.src)
       return ;
 
+    function queryParams (source) {
+      let array = [];
+      for(let key in source) {
```
Comment (jatinkatyal13) [inline app/pods/components/hls-player/component.js]: use `.map()` instead of for loop

Comment (farhan787) [reply app/pods/components/hls-player/component.js]: `source` is an object, map won't work here.

**Comment by farhan787** [comment]
@jatinkatyal13 on installing ember-fetch using `ember install ember-fetch` all the integrity hashes are being removed by yarn.

**Pull request closed without merging**